src/modules/system-settings/application/controllers/__tests__/system-settings.controller.spec.ts

```
@@ -0,0 +1,232 @@
+        },
+        {
+          provide: ImportSettingsUseCase,
+          useValue: {
+            execute: jest.fn(),
+          },
+        },
+      ],
+    }).compile();
+
+    controller = module.get<SystemSettingsController>(SystemSettingsController);
+    getSystemSettingsUseCase = module.get<GetSystemSettingsUseCase>(GetSystemSettingsUseCase);
```

ESLint

[error] Replace `GetSystemSettingsUseCase` with `⏎······GetSystemSettingsUseCase,⏎····`

(prettier/prettier)